src/management/__generated/managers/ActionsManager.ts

```diff
@@ -0,0 +1,810 @@
+/* tslint:disable */
+/* eslint-disable */
+import * as runtime from '../../runtime';
+import type {
+  GetActionVersions200Response,
```

Comment:
Since we're not using the 4x/5x responses, we can drop `"200"` from these types

Reply:
This is also related to the fact that these are defined inline instead of using a ref to components/schemas.